openslides_votecollector/serializers.py

```diff
@@ -41,15 +41,30 @@ class SeatSerializer(ModelSerializer):
     """
     Serializer for openslides_votecollector.model.Seat object.
     """
+    is_active = SerializerMethodField()
+
     class Meta:
         model = Seat
         fields = (
             'id',
             'number',
             'seating_plan_x_axis',
             'seating_plan_y_axis',
+            'is_active',
         )
 
+    def get_is_active(self, obj):
```

Comment:
Are you sure, that a (possible) empty queryset (keypad) is a falsy value (line 58). If not you have to use `exists()`.